.travis.yml

```diff
@@ -46,3 +46,16 @@ script:
   - node ./bin/copy-assets.js --mc ./firefox
   - node ./bin/ci/check-file-sizes.js
   - ./node_modules/.bin/mochii --ci --mc ./firefox --headless devtools/client/debugger/new
+  - ./node_modules/.bin/mochii --ci --mc ./firefox --headless
+    browser_toolbox_debugger.js
```

Comment:
the real name of the test is browser_browser_toolbox_debugger.js (and it will time out until we have a m-c changeset with the fix from https://bugzilla.mozilla.org/show_bug.cgi?id=1432842)